src/engines/kotor/script/functions_action.cpp

```diff
@@ -83,6 +83,31 @@ void Functions::actionCloseDoor(Aurora::NWScript::FunctionContext &ctx) {
 	door->close(0);
 }
 
+void Functions::actionMoveToObject(Aurora::NWScript::FunctionContext &ctx) {
+	Engines::KotOR::Object *caller = ObjectContainer::toObject(ctx.getCaller());
+
+	Engines::KotOR::Object *object = ObjectContainer::toObject(ctx.getParams()[0].getObject());
+	if (!object)
+		// TODO: Use PC if object tag is empty. Not sure if this is correct.
+		object = _game->getModule().getPC();
+
+	float range = ctx.getParams()[2].getFloat();
+
+	float x, y, z;
+	caller->getPosition(x, y, z);
```

Comment:
You're not checking that the caller isn't 0 before dereferencing it here.

In fact, in the lower Taris cantina, tar_m03ae, when clicking on Mission, xoreos crashes here.